app/src/components/Calendar.js

```diff
@@ -37,31 +37,37 @@ const Calendar = (data) => {
     setMonth(month.add(addValue, "month"));
     setDaysInMonth(month.daysInMonth());
   };
+  // const emptyInitialCards = [];
+  // for (let i = 0; i < empty_initial_boxes; i++) {
+  //   emptyInitialCards.push(`<Card key={0${i}}></Card>`);
+  // }
+  // console.log(emptyInitialCards);
   return (
     <section>
-      <button onClick={() => changeMonthValue(-1)}>
-        <i className="bi bi-arrow-left-circle"></i>
-      </button>{" "}
-      {month.format("MMMM YYYY")}{" "}
-      <button onClick={() => changeMonthValue(1)}>
-        <i className="bi bi-arrow-right-circle"></i>
-      </button>
+      <h2>
+        <button onClick={() => changeMonthValue(-1)}>
+          <i className="bi bi-arrow-left-circle"></i>
+        </button>{" "}
+        {month.format("MMMM YYYY")}{" "}
+        <button onClick={() => changeMonthValue(1)}>
+          <i className="bi bi-arrow-right-circle"></i>
+        </button>
+      </h2>
       <CardGroup>
+        {/* {emptyInitialCards.join("")} */}
         {dates.map((date, i) => (
           <DayOfMonth
             date={date}
             key={i}
-            data={data.data.phase[dates[i].date()].phaseName}
-            svg={data.data.phase[dates[i].date()].svg}
+            data={data[date.date()].phaseName}
+            svg={data[date.date()].svg}
```

Comment:
What if you add the `?` you do earlier here? When the page doesn't work, it shows line `62` as the issue, with the same `undefined` thing that happened with the Day View